### ISSUE TITLE
Temp and Humidity Sensors Disabled In Apple Home App

### ISSUE THREAD

**Issue by GioCirque** [naofireblade/homebridge-weather-plus#143]
The temperature and humidity sensors are displayed in the Home App and are not usable as automation triggers.

Is this a setting that needs adjusting, or a limitation of the implementation?

### Available Sensors
![image](https://user-images.githubusercontent.com/5244282/84405301-91bdc580-abcd-11ea-9b7a-adf595ad00ad.png)

### Unavailable Sensors
![image](https://user-images.githubusercontent.com/5244282/84405413-b2861b00-abcd-11ea-844a-d4c61578f7ae.png)

**Comment by chrispoole643** [comment]
Yes: humidity sensor isn't one that triggers occupancy to be able to be used in automations.

Is it possible to treat this as an occupancy trigger too?
